vaadinfrontend/frontend/themes/datamanager/components/card.css

```
@@ -27,6 +27,23 @@
   cursor: pointer;
 }
 
+/* Styles shamelessly stolen from the vaadin primary theme */
+
+.creation-card .button {
+  background-color: var(--lumo-button-primary-background-color, var(--lumo-primary-color));
+  color: var(--lumo-button-primary-color, var(--lumo-primary-contrast-color));
+  font-weight: 600;
+  min-width: calc(var(--lumo-button-size) * 2.5);
+}
+
+.creation-card .button::before {
+  background-color: black;
+}
+
+.creation-card .button:hover::before {
+  opacity: 0.05;
+}
+
```

NIT: Since the creation card only has one button we could skip the definition of an extra class for that via:

```suggestion
.creation-card vaadin-button {
  background-color: var(--lumo-button-primary-background-color, var(--lumo-primary-color));
  color: var(--lumo-button-primary-color, var(--lumo-primary-contrast-color));
  font-weight: 600;
  min-width: calc(var(--lumo-button-size) * 2.5);
}

.creation-card vaadin-button::before {
  background-color: black;
}

.creation-card vaadin-button:hover::before {
  opacity: 0.05;
}
```
Another alternative would be to move all the predefined vaadin-component themes into dedicated classes within a seperate css in a seperate PR so it's reusable 🤔